lib/tasks/users.rake

```diff
@@ -0,0 +1,16 @@
+namespace :users do
+  task :create, [:email, :password] => :environment do |t, args|
+    unless args[:email].present?
+      puts 'Email is required'
+      next
+    end
+
+    unless args[:password].present?
+      puts 'Password is required'
```

Comment:
You can use `abort('Password is required')` in Rake tasks which outputs the message and returns an exit code of 1